scss/c2s/c2s.scss

```diff
@@ -1,5 +1,8 @@
 @import "betahealth/nhsuk";
 
+$callout-border-blue: #40ADD7;
```

Comment:
This looks like duplication, checkout lines 10 and 11 https://github.com/nhsuk/profiles/blob/1dfdc675cfa12e5157b82c055bf66edbf232d63d/scss/c2s/c2s.scss